### ISSUE TITLE
Possible incorrect values and disable multiple choice

### ISSUE THREAD

**Issue by brendanator** [codespell-project/codespell#1193]
Below is a list of words that appear on both the left and right hand sides of the dictionary. I assume that most of these are errors.

I spotted this as I'm using `minimise` (English spelling) and it complained even though it is a correct spelling on the right hand side. Is there an option to disable suggestions when there is more than one choice?

```
$ rg --pcre2 '^([a-z]*)->.*\b\1\b' data/dictionary.txt
1931:backword->backward, backword,
1932:backwords->backwards, backwords,
2445:calculatable->calculatable, calculable,
2951:circularly->circular, circularly,
3252:commend->commend, comment, command,
3253:commends->commends, comments, commands,
4245:copyable->copyable, copiable,
4456:crate->create, crate,
4457:crated->created, crated,
4484:cretaing->cretaing
4488:cristal->crystal, cristal,
4686:dealign->dealing, dealign,
4947:degrate->degrate, degrade,
4948:degrates->degrates, degrades,
7664:followings->followings, following,
7702:forewarded->forewarded, forwarded,
7732:formule->formula, formulas, formule,
7780:fount->fount, found,
8080:gird->grid, gird,
8081:girds->grids, girds,
8506:hove->hove, have, hover, love,
8910:inactivate->inactivate, deactivate,
9410:inly->only, inly,
9984:itteration->itteration
10168:leaded->led, lead, leaded,
10262:lightening->lightening, lightning, lighting,
10326:liszt->list, liszt,
10357:loafing->loading, loafing,
10409:lousily->lousily, loosely,
10613:mater->matter, master, mother, mater,
10731:memorise->memorise, memorize,
10881:minimise->minimise, minimize,
11097:mould->mold, mould, module,
11619:ocorrence->ocorrence
11620:ocorrences->ocorrences
12161:panting->panting, painting,
13399:prosses->process, processes, possess, prosses,
13499:purportive->supportive, purportive,
13628:readd->readd, re-add, read,
13629:readded->readded, read,
13660:ream->ream, stream,
14678:retuned->retuned, returned,
14718:revered->revered, reversed,
14879:sate->state, sate,
15391:sinc->sinc, synch, sync, sink, since,
15394:sincs->sincs, syncs, sinks, since,
15397:singe->singe, single,
15398:singed->signed, singled, singed,
15867:sting->string, sting,
15869:stings->strings, stings,
16286:suppressable->suppressable, suppressible,
16825:therefor->therefore, therefor,
16834:theses->these, thesis, theses,
17240:tread->tread, thread, treat,
17647:unknow->unknown, unknow,
17648:unknows->unknowns, unknows,
18014:valuess->valuess
18207:vise->vice, vise,
18307:wasn->wasn't, was,
18326:webp->webp, web,
18379:whet->when, what, wet, whet,
18431:wight->weight, white, right, wight,
18433:wights->weights, whites, rights, wights,
18467:wit->wit, with,
18475:wither->wither, either, whether, weather,
18516:wont->won't, wont,
18633:zoon->zoom, zoon,
```

**Comment by luzpaz** [comment]
Most are correct but are edgecases.  For example `whet` is a legit word but in many instances it is used incorrectly. Same as `gird` etc...

**Comment by peternewman** [comment]
See also https://github.com/codespell-project/codespell/pull/1142

**Comment by luzpaz** [comment]
Issue with these is that we don't have a great solution for either common misspellings and archaic words that are valid but used seldom. the idea is to maybe have a separate dictionary that a user can opt to use that will check this usecase

**Comment by peternewman** [comment]
> I spotted this as I'm using `minimise` (English spelling) and it complained even though it is a correct spelling on the right hand side.

The British English bit will be #103 too.

**Comment by peternewman** [comment]
This want checking that they've moved into the correct dictionary once #1142 has gone in.

**Comment by peternewman** [comment]
Just reopening for another check as at least some (like memorise look to be en_GB), we should try and check the rest.

**Comment by bl-ue** [comment]
This is no longer a problem (fixed in #1142)

https://github.com/codespell-project/codespell/blob/5a8085ac47e4aeeb3a8983c64355a152ff00ad4c/codespell_lib/tests/test_dictionary.py#L120-L121
